lib/machinegun.rb

```diff
@@ -2,27 +2,27 @@
 require 'filewatcher'
 
 class MachineGun
-  
-  def self.run
+
+  def self.run(interval = nil)
```

Comment:
Optional parenthesis: `def self.run interval = 0.5`

Adding the interval here means it doesn't have to be OR'd later, and documents the default value. I'll probably end up moving this to an option hash eventually so a change in options doesn't change the method signature.